### PR TITLE
add conduwuit

### PR DESCRIPTION
**Pull request by girlbossceo**
To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

Note:
- lowercase `conduwuit` is intentional, as I prefer it over `Conduwuit`.
- we are listed as an independent server at https://matrix.org/ecosystem/servers/ (just in case I get the "why not contribute to upstream?" questions here)
- I am resubmitting this PR as the date mentioned here https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/699#issuecomment-2102322088 has passed 😇

### CODE REVIEW

**Comment by Ki-er** [comment]
There doesnt seem to be releases for this however there are GitHub pages deployments so will wait for others to decide whether these are sufficient. 

If it is decided that they are not, please create a release and wait atleast 4 months before resubmitting this project.

**Comment by girlbossceo** [comment]
Hi @Ki-er, there should be releases here: https://github.com/girlbossceo/conduwuit/releases

Unless you are referring to something else?

**Comment by Ki-er** [comment]
> Hi @Ki-er, there should be releases here: https://github.com/girlbossceo/conduwuit/releases
> 
> Unless you are referring to something else?

I stand corrected, I couldnt find them! thank you for the link.

File: software/conduwuit.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit with tons of new features, many bug fixes, huge performance improvements, quality of life enhancements, moderation tools, and much more!
```
[the remark this answers is not in the document]
Comment (girlbossceo) [reply software/conduwuit.yml]: Added

**Review by kokomo123 — APPROVED** [review]
I shall await other contributor's input, but as far as I can tell, LGTM.  Thank you for your contribution!

File: software/conduwuit.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit with tons of new features, many bug fixes, huge performance improvements, quality of life enhancements, moderation tools, and much more!.
```
Comment (nodiscc) [inline software/conduwuit.yml:3]: The description should explain what the software does

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/conduwuit.yml (hunk 2 above)
Comment (kokomo123) [reply software/conduwuit.yml:3]: It mentions Conduit, yeah but I guess it really should explain what it is. Like a matrix server or something. I see then.

File: software/conduwuit.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit, which is a Matrix chat homeserver.
```
Comment (nodiscc) [inline software/conduwuit.yml:3]:
```suggestion
description: Well-maintained, featureful Matrix chat homeserver (fork of Conduit).
```

https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

**Review by nodiscc — CHANGES_REQUESTED** [review]
Also:

> If you are adding software forked from another active project, please provide/link to a clear list of differences between both.

File: software/conduwuit.yml (hunk 3 above)
Comment (kokomo123) [reply software/conduwuit.yml:3]: https://conduwuit.puppyirl.gay/differences.html @nodiscc

Comment (girlbossceo) [reply software/conduwuit.yml]: Also applied the suggested description.

Comment (nodiscc) [reply software/conduwuit.yml]: https://conduwuit.puppyirl.gay/differences.html is more than fine, thank you, I did not see it at first.

**Review by nodiscc — APPROVED** [review]

**Review by kokomo123 — APPROVED** [review]

**Pull request merged**